src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
```

Comment:
That's the point though, we don't want the `/` line in `crypttab`. `crypttab` unlocking is only ever supposed to take care of other mount points, like `/home`. If your only mount point is `/`, it is correct for `crypttab` to be empty because `/` is already unlocked early on, before `crypttab` kicks in.